internal/artifacts/module_arch.go

```diff
@@ -202,7 +208,7 @@ func checkBuildResultsConflicts(mtaObj *mta.MTA, source, target string, extensio
 		if err != nil {
 			return err
 		}
-		moduleNameResultPathMap[moduleName] = false
+		moduleNameResultPathMap[moduleName] = true
```

Comment:
Can you rename this map to maybe `modulesToPack` (or anything else that doesn't have `resultPath` in its name)?

Reply:
renamed